address_book.rb

```diff
@@ -1,13 +1,22 @@
-class AddressBook
-  def initialize(csv_path)
-
-  end
+require 'csv'
 
-  def print_people
+class AddressBook
+	def initialize(csv_path)
+  	#Read CSV File
+  	@people = CSV.read(csv_path)
```

Comment:
@VictorTomaili o regex'e göre bastırma olayını zaten anlayamadım tam olarak o yüzden hiç beceremedim sanırım. person[0] bastırmadaki hatayı anladım, ama virgül kısmı biraz pürüzlü açıkçası.

@beydogan ve @VictorTomaili teşekkür ederim yardım için. :)